### PR TITLE
Add existing secret configuration for Google SSO

### PR DESCRIPTION
**Pull request by lazy-slackoth**
Now there is no possibility to pass clientId and clientSecret for [Google SSO config](https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L811-L814) from existing secret.
This PR aimed to fix this.

### CODE REVIEW

File: sentry/templates/configmap-sentry.yaml [hunk 1]
```diff
@@ -55,10 +55,10 @@ data:
     ##########
     # Google #
     ##########
-    {{- if .Values.google.clientId }}
-    auth-google.client-id: {{ .Values.google.clientId | quote }}
-    auth-google.client-secret: {{ .Values.google.clientSecret | quote }}
-    {{ end }}
+    {{- if and (.Values.google.clientId) (.Values.google.clientSecret) (not .Values.google.existingSecret) }}
```
Comment (Mokto) [inline sentry/templates/configmap-sentry.yaml:58]: I think you can make it `{{- if and (.Values.google.clientId) (.Values.google.clientSecret) }}` that's enough.

We should pass either of the possibilities

File: sentry/templates/_helper.tpl [hunk 2]
```diff
@@ -533,6 +533,18 @@ Common Sentry environment variables
       name: {{ .Values.github.existingSecret }}
       key: {{ default "client-secret" .Values.github.existingSecretClientSecretKey }}
 {{- end }}
+{{- if and .Values.google.existingSecret }}
```
Comment (Mokto) [inline sentry/templates/_helper.tpl]:
```
{{- if .Values.google.existingSecret }}
```

File: sentry/templates/configmap-sentry.yaml (hunk 1 above)
Comment (lazy-slackoth) [reply sentry/templates/configmap-sentry.yaml:58]: It's just additional step to protect against case where user define _.Values.google.clientId_ and _.Values.google.existingSecret_.

**Comment by vjdhama** [comment]
We are also looking for this.

Is there something I can help with to get this merged?

**Comment by lazy-slackoth** [comment]
Honestly saying I don't understand why MR still in open state, @Mokto could you prompt what else can be fixed?

File: sentry/templates/configmap-sentry.yaml [hunk 3]
```diff
@@ -55,10 +55,10 @@ data:
...
+    auth-google.client-id: {{ .Values.google.clientId }}
+    auth-google.client-secret: {{ .Values.google.clientSecret }}
```
Comment (Mokto) [inline sentry/templates/configmap-sentry.yaml]: Why did you remove `quote` here?

Comment (lazy-slackoth) [reply sentry/templates/configmap-sentry.yaml]: Good that you noticed that, will return

**Review by Mokto — CHANGES_REQUESTED** [review]
Please upgrade the minor version as you're adding a new feature.

**Review by Mokto — APPROVED** [review]

**Comment by Mokto** [comment]
Thanks!
